src/agoradatatools/great_expectations/gx/checkpoints/agora-test-checkpoint.yml

```diff
@@ -0,0 +1,30 @@
+name: agora-test-checkpoint
```

Comment:
@BWMac can you explain what this file is for?

Reply:
Thanks for catching that. It was an additional file generated when I ran the Jupyter Notebook which created an identical checkpoint with a different name.